### PR TITLE
Add the ledger asset

### PR DESCRIPTION
**Pull request by Ray2126**
Test plan:
- Go to docs.centrapay.com - see that the changes are showing as expected

### CODE REVIEW

File: src/content/api/asset-types.mdx [hunk 1]
```diff
@@ -11,19 +11,20 @@ The following table describes the Asset Types supported for payments.
...
+| cca.coke         | Coke tokens                                         | [Token](/api/assets#token-model) | `NZD`                         | ⚡️ 🧪 🚫   |
+| centrapay.nzd    | Centrapay NZD wallet                                | [Money](/api/assets#money-model) | `NZD`                         | ⚡️ 🧪     |
+| centrapay.token  | Centrapay tokens                                    | [Token](/api/assets#token-model) | `NZD`                         | ⚡️ 🧪     |
+| epay             | [EPay NZ giftcards](https://www.giftstation.co.nz/) |                                  | `NZD`                         | ⚡️ 🧪     |
+| farmlands.nzd    | [Farmlands](https://www.farmlands.co.nz/)           | [Money](/api/assets#money-model) | `NZD`                         | ⚡️ 🧪 💸 💼 |
+| paypal           | [PayPal](https://www.paypal.com/)                   |                                  | `USD`                         | ⚡️ 🧪 💸   |
+| centrapay.ledger | Centrapay Ledgers                                   |                                  | `NZD`                         | ⚡️ 🧪     |
```
Comment (seliwe) [inline src/content/api/asset-types.mdx:23]: Change: under Category we should include a link to the Ledger Model, same as tokens

File: src/content/api/assets.mdx [hunk 2]
```diff
@@ -160,6 +160,53 @@ Tokens have the following attributes along with the base asset fields.
 
 ---
 
+## Ledger Model <Badge type="experimental"/>
+
+Ledgers are a container for tokens. Ledgers contain a balance of the number of tokens the account has from a specific collection. They can be spent and sent in arbitrary amounts.
```
Comment (seliwe) [inline src/content/api/assets.mdx:165]: Suggestion: `A ledger is a single asset that maintains a balance of multiple tokens belonging to an account. All tokens in a ledger come from a specific collection. Multiple tokens can be spent or sent at once from the ledger.`

File: src/content/api/assets.mdx [hunk 3]
```diff
@@ -160,6 +160,53 @@ Tokens have the following attributes along with the base asset fields.
...
+
+Ledgers have the following attributes along with the base asset fields.
+
+<Properties>
+  <Property name="collectionId" type="string">
+    The [token collection](/api/tokens#token-collection-model) that will govern the branding and redemption rules for the token.
```
Comment (seliwe) [inline src/content/api/assets.mdx:171]: Suggestion: `The token collection that all tokens in the ledger belong to`

File: src/content/api/assets.mdx [hunk 4]
```diff
@@ -160,6 +160,53 @@ Tokens have the following attributes along with the base asset fields.
...
+  <Property name="tokenValue" type="monetary">
+    The value of each token in the ledger. Taken from the `maxValue` property on the token collection.
+  </Property>
+
+  <Property name="createdBy" type="crn">
+    The identity that created the ledger.
+  </Property>
+
+  <Property name="expiresAt" type="timestamp">
+    The date when the asset expires.
```
Comment (seliwe) [inline src/content/api/assets.mdx:187]: Suggestion: `The date when the ledger expires`

File: src/content/api/asset-types.mdx (hunk 1 above)
Comment (Ray2126) [reply src/content/api/asset-types.mdx]: Yep nice one!

**Review by seliwe — APPROVED** [review]

**Pull request merged**